src/app/containers/RecentAudioEpisodes/index.jsx

```diff
@@ -75,7 +75,15 @@ const RecentAudioEpisodes = ({ masterBrand, episodes }) => {
   const audioLabel = pathOr('Audio', ['media', 'audio'], translations);
   const getUrl = episodeId =>
     '/'.concat(
-      [service, variant, masterBrand, episodeId].filter(Boolean).join('/'),
+      [
+        service,
+        variant,
+        brandId ? 'podcasts' : '',
```

Comment:
Is it maybe worth us passing `metadata: { type: 'Podcast' },` down to this component and using that for this logic? 

It isn't obvious from the code that only podcasts will have a `brandId` whereas using a value that would === 'Podcast' is quite explicit.